rust/cedar-policy-mcp-schema-generator/src/lib.rs

```diff
@@ -14,17 +14,10 @@
  * limitations under the License.
  */
 
-mod description;
-mod err;
-mod schema_generator;
+mod generator;
```

Comment:
While we're making breaking changes, should we _also_ make this a `pub mod` and remove the `pub use` exports below? Effectively making the generator-related items live in the `generator` namespace instead of the root namespace?  I guess the name of the crate is `cedar-policy-mcp-schema-generator` so it probably makes sense to have the generator stuff in the root namespace